src/css/base.css

```diff
@@ -64,11 +64,20 @@ pre {
 
 code {
   background-color: #0074e00e;
-  font-size: var(--font-small);
+  /* font-size: var(--font-small); */
+  font-size: inherit;
```

Comment:
The font size is going to inherit by default, so as long as you remove the explicit override, it will just work.

Reply:
This was needed to address code font in headings, paragraphs, etc.

Reply:
That's because the font size is being set in src/css/base.css. If you don't want the font size to be different than the font size of the parent element, it's best just to remove this altogether. That way you don't need to override it in multiple places.

Reply:
Hi @mojavelinux ,
Thanks for the clarification, Now I removed font size from `code`.